feat: calculate water coverage

connorholliday5 opened the pull request.

connorholliday5 committed:
feat: calculate water coverage

connorholliday5 committed:
refactor: delete file

eldu commented:
* Take a look at python docstrings
* Add a docstring to the top of the file (above imports) to explain what the file is
* I think all of the code at the bottom should be in a main function - https://mostlypython.substack.com/p/do-you-really-need-a-main-function

connorholliday5 committed:
changed comments to doc strings
added doc string to explain file on top
added a main function so the code doesnt run continuous

connorholliday5 committed:
chnaged name to ice_shelf_water_extent

connorholliday5 committed:
Update README.md

connorholliday5 committed:
Merge branch 'main' into feat-water-coverage
